src/MeetingsSDKAdapter.test.js

```diff
@@ -211,6 +212,46 @@ describe('Meetings SDK Adapter', () => {
     });
   });
 
+  describe('fetchMeetingTitle()', () => {
+    beforeEach(() => {
+      global.console = {
+        warn: jest.fn(),
+      };
+    });
+
+    test('returns meeting sipUri as title', async () => {
+      const meetingTitle = 'sipUri';
+
+      expect(await meetingSDKAdapter.fetchMeetingTitle(meetingTitle)).toEqual(meetingTitle);
+    });
+
+    test('returns person displayName if the destination is a personID', async () => {
+      const meetingTitle = 'PEOPLE';
```

Comment:
Can you rename for clarity?
```suggestion
      const meetingTitle = 'HYDRA_PEOPLE_ID';
```